orbitx/Vpython.py

```diff
@@ -0,0 +1,31 @@
+import vpython
+import signal
+
+vpython_object = vpython
+container = [vpython_object]
+closed = False
+
+def vpython_version_check() -> None :
+    print("version check is executed")
+    ctrl_c_handler: function = signal.getsignal(signal.SIGINT)
+    if vpython_object.__version__ == '7.4.7':
```

Comment:
putting vpython in its own module does not make sense, since this code will be run every time it's imported by other modules instead of just once.

But also, this version check is just too expensive to maintain if we have to do a workaround hack like this `vp.py` file. We should just get ride of this version check completely, since it was ugly to begin with, and change the line in `requirements.txt` to be
`git+https://github.com/BruceSherwood/vpython-jupyter`
instead of
`vpython>=7.4.7`
since vpython master is currently newer than 7.4.7

Reply:
I didn't know that you think the version check code is unnecessary. You needed to tell me we can get rid of it. Also the version_check will be executed just once since we use vp().

Reply:
It was only somewhat necessary, but per the comments in my other PR I have submitted a patch to vpython maintainers to fix the problem. It was never an entirely critical problem to begin with, and when I read through this PR I realized that the necessary code to keep this verion-specific patch in the code was more trouble than it was worth. The quality of the code should not suffer because of it, and if the version-specific code forces you to make a non-pythonic solution like an entire wrapping module `Vpython.py`, it's more trouble than it's worth and should be removed.

We can keep it in, but I think it's less work for all of us if we just drop it now instead of maintaining it.